src/arch-syscall-validate

```diff
@@ -306,6 +306,50 @@ function dump_lib_aarch64() {
 	dump_lib_arch aarch64 | mangle_lib_syscall aarch64
 }
 
+#
+# Dump the loongarch64 syscall table
+#
+# Arguments:
+#     1    path to the kernel source
+#
+#  Dump the architecture's syscall table to stdout.
+#
+function dump_sys_loongarch64() {
+	local sed_filter=""
+
+	sed_filter+='s/__NR3264_fadvise64/223/;'
+	sed_filter+='s/__NR3264_fstatat//;'
+	sed_filter+='s/__NR3264_fcntl/25/;'
+	sed_filter+='s/__NR3264_fstatfs/44/;'
+	sed_filter+='s/__NR3264_fstat//;'
```

Comment:
Since loongarch64 doesn't define either `__ARCH_WANT_NEW_STAT` or `__ARCH_WANT_STAT64` this line isn't strictly necessary is it?

Reply:
Yes, you are right.